diamm/templates/website/search/results/result.jinja2

```diff
@@ -28,6 +28,9 @@
 
 <div class="search-result">
     <h4>
+        {% if result.public_images_b and result.type == "source" %}
```

Comment:
You should reverse these conditions; Python evaluates the first statement and then the second. Since there will only be a `public_images_b` field on `source` objects, you can check first if it's a source, and then check if the images are public.